roles/ovirt.storages/tasks/main.yml

```diff
@@ -2,14 +2,18 @@
 #################################################
 # Storages
 #################################################
+- name: Get list of hosts
+  set_fact:
+    storage_hosts: "{{ lookup('vars', hosts_var_name) }}"
```

Comment:
Hmm, wouldn't it be better to use that approach for the whole infra role? I mean having a pre_task which will define correct hosts fact (for example `ov_ifr_hosts`) and use this fact in all place within the role.

Reply:
Will do